### ISSUE TITLE
Having `rack` gem is not a guarantee for having `rackup` executable (since Rack 3)

### ISSUE THREAD

**Issue by dentarg** [heroku/heroku-buildpack-ruby#1356]
Spotted this when creating #1355

https://github.com/heroku/heroku-buildpack-ruby/blob/104fe3a374e07a8f3723f110c2148d57ecb9ee79/lib/language_pack/rack.rb#L23-L32

Since Rack 3, that code needs another guard on the `rackup` gem: https://github.com/rack/rack/blob/v3.0.3/UPGRADE-GUIDE.md#binrackup-was-moved-to-a-separate-gem

**Comment by schneems** [comment]
Good catch. What do you think would be a good default here when there is no rackup gem? Specify no web default?

**Comment by dentarg** [comment]
Sounds resonable
